js/busy-indicators/busy-indicators.scss

```diff
@@ -0,0 +1,181 @@
+/*
+  CSS for shiny.ui.busy_indicators module.
+
+  The general idea is to leverage the `recalculating` class that shiny.js adds to each
+  output when it's recalculating and add a spinner to the `::after` pseudo-element.
+*/
+
+/* "Built-in" spinner types */
+/* Part of the reason why they're all here is so we can embed the SVGs in the CSS */
```

Comment:
Make it explicit in the comment that esbuild will embed them in the CSS (not just that the svg's can be embedded)